public/js/analytics_dnt-helper.js

```diff
@@ -1,3 +1,6 @@
+/*eslint-disable no-var */
```

Comment:
Why was this file moved/renamed?
It will probably affect the .eslintignore file since we were previously ignoring this file, IIRC.

Reply:
and here I was wondering why eslint had suddenly taken an interest in this file. It was moved and renamed for the purpose of concatenating the files in the same order they're linked in the website. They're is probably a cooler way to handle this.

Reply:
> They're is probably a cooler way to handle this.

I imagine we could use **webpack** or something to create a fancy bundle, but this all may be too risky to add for launch.

Reply:
and by <i>They're</i> I meant <i>There</i>. How embarrassing.